distributed/scheduler.py

```diff
@@ -1940,7 +1943,19 @@ def _transition(
                     parent._tasks[ts._key] = ts
                 for plugin in list(self.plugins):
                     try:
-                        plugin.transition(key, start, finish2, *args, **kwargs)
+                        plugin.transition(
+                            key,
+                            start,
+                            finish2,
+                            *args,
+                            status=status,
+                            thread=thread,
+                            metadata=metadata,
+                            worker=worker,
+                            startstops=startstops,
+                            text=text,
+                            **kwargs,
+                        )
```

Comment:
I don't think this is a dealbreaker but that _could_ be perceived as a breaking change, couldn't it? At least this is a subtle change in the required signature. So far we document this as `*args, **kwargs`, see https://distributed.dask.org/en/latest/plugins.html#distributed.diagnostics.plugin.SchedulerPlugin.transition

Reply:
I don't think so. These are already passed as keyword arguments and we are continuing to pass them that way

That said, Idk if this is how things will shake out yet. Things are still changing here. IOW I'm not sure this is worth reviewing yet

Reply:
+1 to what John said.  I think that this is a no-op if you consider the extraction of these from kwargs above.